packages/create-mc-app/src/tasks/download-template.ts

```diff
@@ -26,7 +26,9 @@ function downloadTemplate(options: TCliTaskOptions): ListrTask {
       );
       const templateFolderPath = path.join(
         clonedRepositoryPath,
-        'application-templates',
+        options.applicationType === ApplicationTypes.CustomApplication
```

Comment:
We update values in a different config file depending on the type of application we want to create.